mgmtsystem_action/models/mgmtsystem_action.py

```diff
@@ -184,14 +174,14 @@ def write(self, vals):
                     body=' %s ' % (_('Action cancelled on ') +
                                    fields.Datetime.now())
                 )
-        return super(MgmtSystemAction, self).write(vals)
+        return super(MgmtsystemAction, self).write(vals)
 
+    @api.multi
     def send_mail_for_action(self, action, force_send=True):
-        """Set a document state as draft and notified the reviewers."""
         template = self.env.ref(
             'mgmtsystem_action.email_template_new_action_reminder')
-        for action in self:
-            template.send_mail(action.id, force_send=force_send)
+        self.env['mail.template'].browse(template.id).send_mail(
+            action.id, force_send=force_send)
```

Comment:
@sbidoul Hi
The mails are send on record creation, there is no need for a looping.
I actually wanted to preserve the looping since i wasnt sure about its purpose... 

So i did some extensive testin just to figure out that the looping prevented the mails from beeing send on record creation